Sasquatch/Sasquatch/ViewControllers/Sections/CommonSchemaPropertiesTableSection.swift

```diff
@@ -60,8 +60,22 @@ class CommonSchemaPropertiesTableSection : SimplePropertiesTableSection {
     }
   }
   
+  // Note down current target for using in End edit events
```

Comment:
Another nit: Please add `.` at the end of your sentences in comments.

Reply:
Ok, added the dot